core/services/directrequest/delegate.go

```diff
@@ -104,7 +104,8 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.Service, error) {
 		db:                       d.db,
 		pipelineORM:              d.pipelineORM,
 		job:                      jb,
-		mbLogs:                   utils.NewMailbox(50),
+		mbOracleRequests:         utils.NewMailbox(50),
```

Comment:
I think 50 is way too low, we recently increased the VRF to 100k and could do the same here. Imagine we replay a few thousand blocks on matic, its quite possible you could quickly queue up a few thousand requests.

Reply:
Doesn't seem like it would hurt to set it to something massive like 100k its just a sanity limit anyway

Reply:
created a separate PR: https://github.com/smartcontractkit/chainlink/pull/4995